src/app/subscribe-events.ts

```diff
@@ -62,17 +63,27 @@ export function subscribeToComponentEvents(wrapperCtx: any) {
 
             let highlightQuery: any = undefined;
```

Comment:
Not sure why upstream used here "any", can you check if it's simply `QueryParam | undefined`? (and review any errors that TS will bring up with type, which will be probably bugs anyway)